### PR TITLE
Penumbra user agent support hints & Babel changes for IE11

### PR DESCRIPTION
**Pull request by eligrey**
This PR adds user agent support hints to Penumbra & aims to change our webpack & Babel config to better-support IE11.

This doesn't change any webpack settings yet, so IE11 will still get some errors. It looks like we're still generating bundles using the unsupported `Symbol` API.

TODO:

- [x] Detect Penumbra UA support
- [x] ~~Remove/polyfill Symbol through Babel~~
- [x] ~~Probably more Babel changes~~

Parent issue: https://github.com/transcend-io/main/issues/3193

https://github.com/transcend-io/main/issues/3213

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -400,6 +400,28 @@ if (!self.penumbra) {
...
+
+```ts
+if (penumbra.supported() > penumbra.supported.levels.possible) {
+  // penumbra is partially or fully supported
+}
+
+/** penumbra.supported.levels - Penumbra user agent support levels */
+enum PenumbraSupportLevel {
+  /** Old browser where Penumbra does not work at all */
+  none = -0,
```
Comment (michaelfarrell76) [inline README.md:415]: what is -0?

**Comment by michaelfarrell76** [comment]
should rebase, i updated webpack configs

File: README.md (hunk 1 above)
Comment (eligrey) [reply README.md:415]: Same as zero, but theoretically detectably different. Support levels `none` & `possible` should be treated mostly the same.

**Comment by eligrey** [comment]
I don't think the codecov check should be failing. This should be fine to merge.

**Comment by bencmbrook** [comment]
Yeah dw about codecov.

Anything we want to do for .encrypt? Not a huge deal.. but there seems to be separate criteria for that?

**Comment by eligrey** [comment]
Ready to merge 👍

**Comment by michaelfarrell76** [comment]
bumped to minor change instead of patch

**Pull request merged**